src/api/render-node.ts

```diff
@@ -74,16 +73,15 @@ export function renderNodeManagement(snapshot: StateSnapshot, config: ServiceCon
 
 // helpers
 
-function getSigner(snapshot: StateSnapshot, config: ServiceConfiguration) {
+function getSigner(snapshot: StateSnapshot) {
   const version = snapshot.CurrentImageVersions['main']['signer'];
   if (!version) return undefined;
 
   return {
     InternalPort: 7777,
     Disabled: false,
     DockerConfig: {
-      Image: `${config.DockerNamespace}/signer`,
-      Tag: version,
+      ...parseImageTag(version),
```

Comment:
nice refactor, but that would mean we no longer have explicit image name checking (for example for signer -> Image: `namespace/signer`). I don't know if it's important here because already taking the version from CurrentImageVersions[main][signer].. anyway to be explicit u can pass the expected name to `parseImageTag`